docker-compose.yml

```diff
@@ -24,6 +24,20 @@ services:
       - static_volume:/app/experimenter/served/
     command: bash -c "/app/bin/wait-for-it.sh kinto:8888 -- python bin/setup_kinto.py;/app/bin/wait-for-it.sh db:5432 -- python /app/manage.py collectstatic --noinput;python /app/manage.py runserver 0:7001"
 
+  yarn-nimbus-ui:
+    image: app:build
+    env_file: .env
+    environment:
+      - CHOKIDAR_USEPOLLING=true
+    tty: true
+    ports:
+      - "3000:3000"
+    volumes:
+      - ./app:/app
+      - /app/node_modules/
+      - /app/experimenter/nimbus-ui/node_modules/
+    command: bash -c "yarn workspace @experimenter/nimbus-ui start"
```

Comment:
FWIW, I just tried this locally and it seems to hack around the console clearing nonsense:

```diff
diff --git a/docker-compose.yml b/docker-compose.yml
index 9cd20549..1ba50734 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -36,7 +36,7 @@ services:
       - ./app:/app
       - /app/node_modules/
       - /app/experimenter/nimbus-ui/node_modules/
-    command: bash -c "yarn workspace @experimenter/nimbus-ui start"
+    command: bash -c "FORCE_COLOR=true yarn workspace @experimenter/nimbus-ui start | cat"
 
   yarn-core:
     image: app:build
```